docs/en/ecosystem_release/starrocks-kubernetes-operator.md

```diff
@@ -40,9 +40,62 @@ For example, the download URL for kube-starrocks chart v1.8.6 is:
 
 ## Release notes
 
-## 1.8
+### 1.9
 
-### 1.8.6
+#### 1.9.1
+
+**Improvements**
+
+- **[Helm Chart]** The operator will not create PersistentVolumeClaim (PVC) for log storage when you set `logStorageSize` to `0`.  [#398](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/398)
+- **[Operator]** The operator can detect the values of`mountPath` and `name` in `storageVolumes` are duplicated. An error is returned when duplicate values exist. [#388](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/388)
```

Comment:
The operator can detect the values of`mountPath` and `name` in `storageVolumes` are duplicated. -> The operator can detect whether the values of`mountPath` and `name` in `storageVolumes` are duplicated.